src/main/resources/citizen/templates/document/claimantDirectionsQuestionnaire.html

```diff
@@ -383,38 +426,43 @@ <h2 class="heading-medium">
     </td>
     <td>
       <div>
-        {{ hearingContent.witness.selfWitness }}
+        {{ hearingContent.witness.selfWitness | capitalize }}
       </div>
     </td>
 
   </tr>
-  <tr>
-    <td>
-      <div>
-        Other witnesses
-      </div>
-    </td>
-    <td>
-      <div>
-        {{ hearingContent.witness.noOfOtherWitness }}
-      </div>
-    </td>
 
-  </tr>
+  {% if hearingContent.witness.noOfOtherWitness is not empty %}
```

Comment:
this can't be empty. it is a Yes/No question so will always have an answer

Reply:
the way this is stored in backend, its not Yes/No. It's an integer which store number of other witnesses. So, this is right.